### PR TITLE
Version Upgrade for ODFE 1.13 Release

### PR DESCRIPTION
**Pull request by davidcui-amzn**
*Issue #, if available:*
N/A
*Description of changes:*
* Update 1.13 release notes
* Bump version in workflows to 1.13.0.0
* Bump ES version to 7.10.2

Query workbench bump will come in separate PR 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## 20201-2-3 Version 1.13.0.0
+
+### Features
+* Added keywords option as alias identifier in SQL parser [#866] (https://github.com/opendistro-for-elasticsearch/sql/pull/866)
```
Comment (joshuali925) [inline release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md:4]: writing it this way markdown will render url as text. could you follow the same format as previous release notes? example from 1.12 release notes:
```
* Add count() support for PPL ([#894](https://github.com/opendistro-for-elasticsearch/sql/pull/894))
```

**Review by joshuali925 — COMMENTED** [review]
Could you also bump these please. I think in 7.10.2 kibana's node version is upgraded, you might also need to change node version in workbench's package.json
```
.github/workflows/sql-workbench-release-workflow.yml
10:  OD_VERSION: 1.12.0.0

.github/workflows/sql-workbench-test-and-build-workflow.yml
7:  OD_VERSION: 1.12.0.0

workbench/kibana.json
3:  "version": "1.12.0.0",

sql-cli/src/odfe_sql_cli/__init__.py
15:__version__ = "1.12.0.0"

sql-jdbc/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/internal/Version.java
22:    Current(1, 12, 0, 0);

sql-odbc/src/CMakeLists.txt
81:set(DRIVER_PACKAGE_VERSION "1.12.0.0")
82:set(DRIVER_PACKAGE_VERSION_COMMA_SEPARATED "1,12,0,0")

sql-odbc/src/TableauConnector/odfe_sql_odbc/manifest.xml
3:<connector-plugin class='odfe_sql_odbc' superclass='odbc' plugin-version='1.12.0.0' name='SQL' version='18.1' min-version-tableau='2020.1'>

sql-odbc/src/TableauConnector/odfe_sql_odbc_dev/manifest.xml
3:<connector-plugin class='odfe_sql_odbc_dev' superclass='odbc' plugin-version='1.12.0.0' name='Open Distro for Elasticsearch SQL ODBC DEV' version='18.1' min-version-tableau='2020.1'>

workbench/package.json
3:  "version": "1.12.0.0",

.github/workflows/sql-workbench-release-workflow.yml
34:          ref: 7.10.0

.github/workflows/sql-workbench-test-and-build-workflow.yml
22:          ref: v7.10.0

workbench/kibana.json
4:  "kibanaVersion": "7.10.0",

workbench/package.json
9:    "version": "7.10.0",

workbench/.cypress/integration/ui.spec.js
172:          'kbn-version': '7.10.0',
```

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md (hunk 1 above)
Comment (davidcui-amzn) [reply release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md:4]: Ack, misread the formatting, will change

**Comment by davidcui-amzn** [comment]
> Could you also bump these please. I think in 7.10.2 kibana's node version is upgraded, you might also need to change node version in workbench's package.json
> 
> ```
> .github/workflows/sql-workbench-release-workflow.yml
> 10:  OD_VERSION: 1.12.0.0
> 
> .github/workflows/sql-workbench-test-and-build-workflow.yml
> 7:  OD_VERSION: 1.12.0.0
> 
> workbench/kibana.json
> 3:  "version": "1.12.0.0",
> 
> sql-cli/src/odfe_sql_cli/__init__.py
> 15:__version__ = "1.12.0.0"
> 
> sql-jdbc/src/main/java/com/amazon/opendistroforelasticsearch/jdbc/internal/Version.java
> 22:    Current(1, 12, 0, 0);
> 
> sql-odbc/src/CMakeLists.txt
> 81:set(DRIVER_PACKAGE_VERSION "1.12.0.0")
> 82:set(DRIVER_PACKAGE_VERSION_COMMA_SEPARATED "1,12,0,0")
> 
> sql-odbc/src/TableauConnector/odfe_sql_odbc/manifest.xml
> 3:<connector-plugin class='odfe_sql_odbc' superclass='odbc' plugin-version='1.12.0.0' name='SQL' version='18.1' min-version-tableau='2020.1'>
> 
> sql-odbc/src/TableauConnector/odfe_sql_odbc_dev/manifest.xml
> 3:<connector-plugin class='odfe_sql_odbc_dev' superclass='odbc' plugin-version='1.12.0.0' name='Open Distro for Elasticsearch SQL ODBC DEV' version='18.1' min-version-tableau='2020.1'>
> 
> workbench/package.json
> 3:  "version": "1.12.0.0",
> 
> .github/workflows/sql-workbench-release-workflow.yml
> 34:          ref: 7.10.0
> 
> .github/workflows/sql-workbench-test-and-build-workflow.yml
> 22:          ref: v7.10.0
> 
> workbench/kibana.json
> 4:  "kibanaVersion": "7.10.0",
> 
> workbench/package.json
> 9:    "version": "7.10.0",
> 
> workbench/.cypress/integration/ui.spec.js
> 172:          'kbn-version': '7.10.0',
> ```

Was originally planning to post a separate PR for workbench but will add the other files and add workbench to this PR if the changes are minimal

**Review by joshuali925 — APPROVED** [review]
LGTM

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* Keep development doc sync with latest code ([#961](https://github.com/opendistro-for-elasticsearch/sql/pull/961))
+* Update ODBC documentation ([#1012](https://github.com/opendistro-for-elasticsearch/sql/pull/1012))
+
+### Maintenance
+* Fix workbench issue in backported 1.11 branch: error message cannot display ([#943] (https://github.com/opendistro-for-elasticsearch/sql/pull/943))
+* Fix URI Encoding in 1.12 ([#955](https://github.com/opendistro-for-elasticsearch/sql/pull/955))
+
+### Breaking Changes
+* Refine PPL head command syntax ([#1022] (https://github.com/opendistro-for-elasticsearch/sql/pull/1022))
+* Disable access to the field keyword in the new SQL engine ([#1025] (https://github.com/opendistro-for-elasticsearch/sql/pull/1025))
```
Comment (joshuali925) [inline release-notes/opendistro-for-elasticsearch-sql.release-notes-1.13.0.0.md:62]: could you also move breaking changes to the top, above features

**Review by chloe-zh — APPROVED** [review]

**Pull request merged**